pvlib/albedo.py

```diff
@@ -0,0 +1,103 @@
+        'clear_water_frequent_whitecaps', 'green_water_ripples_up_to_2.5cm',
+        'muddy_water_no_waves'.
+
+    Returns
+    -------
+    albedo : numeric
+        Albedo for inland water bodies.
+
+    Notes
+    -----
+    The equation for calculating the albedo :math:`\rho` is given by
+
+    .. math::
+       :label: albedo
+
+        \rho = c^{(r sin(\alpha) + 1)}
+
+    Inputs to the model are the water color coefficient :math:`c` [-], the
+    water wave roughness coefficient :math:`r` [-] and the solar elevation
+    :math:`\alpha` [degrees]. Parameters are provided in [1]_ , and are coded
+    for convenience in :data:`~pvlib.albedo.WATER_COLOR_COEFFS` and
+    :data:`~pvlib.albedo.WATER_ROUGHNESS_COEFFS`. The values of these
+    coefficients are experimentally determined.
+
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
+    | Surface and condition                                         | Color coefficient (:math:`c`) | Wave roughness coefficient (:math:`r`) |
+    +===============================================================+===============================+========================================+
+    | Clear water no waves                                          | 0.13                          | 0.29                                   |
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
+    | Clear water ripples up to 2.5 cm                              | 0.16                          | 0.70                                   |
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
+    | Clear water ripples larger than 2.5 cm (occasional whitecaps) | 0.23                          | 1.23                                   |
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
+    | Clear water frequent whitecaps                                | 0.30                          | 2.00                                   |
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
+    | Green water ripples up to 2.5cm                               | 0.22                          | 0.70                                   |
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
+    | Muddy water no waves                                          | 0.19                          | 0.29                                   |
+    +---------------------------------------------------------------+-------------------------------+----------------------------------------+
```

Comment:
```suggestion
    +------------------------+-------------------+-------------------------+
    | Surface and condition  | Color coefficient | Wave roughness          |
    |                        | (:math:`c`)       | coefficient (:math:`r`) |
    +========================+===================+=========================+
    | Clear water, no waves  | 0.13              | 0.29                    |
    +------------------------+-------------------+-------------------------+
    | Clear water, ripples   | 0.16              | 0.70                    |
    | up to 2.5 cm           |                   |                         |
    +------------------------+-------------------+-------------------------+
    | Clear water, ripples   | 0.23              | 1.25                    |
    | larger than 2.5 cm     |                   |                         |
    | (occasional whitecaps) |                   |                         |
    +------------------------+-------------------+-------------------------+
    | Clear water,           | 0.30              | 2.00                    |
    | frequent whitecaps     |                   |                         |
    +------------------------+-------------------+-------------------------+
    | Green water, ripples   | 0.22              | 0.70                    |
    | up to 2.5cm            |                   |                         |
    +------------------------+-------------------+-------------------------+
    | Muddy water, no waves  | 0.19              | 0.29                    |
    +------------------------+-------------------+-------------------------+
```

Reply:
This alternative form should render the same in the HTML (assuming I have not made a mistake), but is much nicer to look at in IDEs and such.